java/spring-framework/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.spring-framework</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>benchmark</name>
+    <description>Demo project for Spring Framework</description>
+    <packaging>war</packaging>
+
+    <parent>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>5.2.0.RELEASE</version>
```

Comment:
could you use a less restict constraint
```xml
<version>[5.2,5.3)</version>
```
to ensure that we stick on **5.2** :+1: 

PS : I've also notice that it is not the case for `spring-boot`

Reply:
Done